hack/arktos-cni.rc

```diff
@@ -158,6 +164,25 @@ install_arktos_containerd() {
     fi
 }
 
+install_mizar_arktos_containerd () {
```

Comment:
thanks for crafting this to ensure the custom containerd proper installation. kudos.